src/parachain/refund.ts

```diff
@@ -38,10 +53,24 @@ export interface RefundAPI {
      * @returns A refund request object
      */
     getRequestByIssueId(issueId: H256): Promise<RefundRequestExt>;
+    /**
+     * Set an account to use when sending transactions from this API
+     * @param account Keyring account
+     */
+    setAccount(account: AddressOrPair): void;
 }
 
-export class DefaultRefundAPI implements RefundAPI {
-    constructor(private api: ApiPromise, private btcNetwork: Network) { }
+export class DefaultRefundAPI extends DefaultTransactionAPI implements RefundAPI {
+    constructor(api: ApiPromise, private btcNetwork: Network, private electrsAPI: ElectrsAPI, account?: AddressOrPair) {
+        super(api, account);
+    }
+
+    async execute(requestId: string, btcTxId?: string, merkleProof?: Bytes, rawTx?: Bytes): Promise<void> {
+        const parsedRequestId = this.api.createType("H256", "0x" + requestId);
+        [merkleProof, rawTx] = await getTxProof(this.electrsAPI, btcTxId, merkleProof, rawTx);
```

Comment:
same as on issue and redeem -> check if user submitted btxTxId and make merkleProof and rawTx getter optional